doc/ref_model/chapters/chapter05.md

```diff
@@ -303,23 +355,34 @@ The configurations specified in here will be used in specifying the actual hardw
 <a name="5.4.3.1"></a>
 #### 5.4.3.1 NIC configurations
 
+| Reference | Feature | Description | Basic Type | Network Intensive |
+|---------------------|-----------|---------------------------|--------|--------|
+| nfvi.hw.nic.cfg.001 | NIC Ports | Total Number of NIC Ports available in the host | 4 | 4 |
```

Comment:
IN the RM are we using "ports" or "connection points"?

Reply:
@pgoyal01 your reviews here are not related to the purpose of the PR. If it OK with you, I will stick to the scope of the PR if possible.